new-starter-guide/2016-03-15-new-starter-to-origami.md

```diff
@@ -0,0 +1,134 @@
+
+### FT.com
+
+Check your email to see if you’ve already been registered for the free subscription to the [ft.com](https://www.ft.com/). If not, sign-up for an account on ft.com then ask `corporate.support@ft.com` to upgrade it to a free subscription.
+
+## Development tools
+
+### Environment setup
+
+Before you start installing anything, it is a good idea to do this (with admin privileges) to ensure that <abbr title="Node Package Manager">npm</abbr> is given permissions to install globally on your machine.
+
+```
+chown -R $USER /usr/local
+```
+
+Almost all Origami applications rely on the following tools to be installed globally on development machines. Please ensure both of these are installed successfully onto your development environment first.
+
+#### Install the following:
+
+- [Node.js](https://nodejs.org/)
+  - We recommend installing a [Node Version Manager](https://github.com/creationix/nvm) to manage different local node versions.
+- [Git](https://git-scm.com/)
+- [Heroku Toolbelt](https://toolbelt.heroku.com/)
+- [Bower](http://bower.io/)
+- [Ruby](http://www.ruby-lang.org/)
+  - We recommend installing a Ruby Version Manager to manage different ruby versions. There are two choices here. You can go either [Ruby Version Manager](https://rvm.io/) or [Rbenv](https://github.com/rbenv/rbenv).
+  - This is for the Ruby SCSS-Lint gem which is a part of <abbr title="Origami Build Tools">OBT</abbr>.
+
+#### Work with our ecosystem of components
+
+You will need to point Bower at the Origami registry. Running the following in a <abbr title="Command Line Interface">CLI</abbr> will do this for you.
+
+```
+[ -e ~/.bowerrc ] || echo '{ "registry": { "search": [ "http://registry.origami.ft.com", "https://bower.herokuapp.com" ] } }' > ~/.bowerrc
+```
+
+You can join all the Origami applications on Heroku by checking out this [shared Google Sheets](https://docs.google.com/a/ft.com/spreadsheets/d/1xk1tyn60ZCmLk1I39Dot-08c9pBJeeX3g9MDENDqjKk/edit?usp=drive_web).
+
+
+#### Issues with installing/getting access
+
+If you experience any problems, then please take a look at Next's [troubleshooting guide](http://financial-times.github.io/next/docs/developer-guide/troubleshooting/). If your problem is not included in the troubleshooting list and you manage to solve it, then please add it to the list.
+
+### Peer review and pull requests
+
+1. For more significant changes always ask for a peer review by posting a link on the relevant channel on slack or cc-ing one or more developers in GitHub.
+
+2. **If no-one reviews your code after a while**
+  - Go and ask somebody directly to do so.
+  - Stand next to them tutting and checking your watch if necessary.
+  - Review it yourself - the time spent waiting will have given you fresh eyes. If you feel confident everything’s OK go ahead and merge.
```

Comment:
Perhaps remove the bullet point steps around this, and write up something along the lines of "all work should go through peer review, and depending on the size of the change should be seen by one or more member of the team?" - written up a bit nicer though? Is that any help @j-mes?